shared/rakefiles/ci.rake

```diff
@@ -53,7 +53,7 @@ task :destroy_hard_and_deploy_ci => [:set_vars_ci] do
   Rake::Task["destroy_hard"].invoke
   begin
     Rake::Task["deploy"].invoke
-    Rake::Task["test_preferences"].invoke
+    Rake::Task["test_preferences_read"].invoke
```

Comment:
I think it is safe to include `test_preferences_write` here – this task is being executer only against ephemeral environments, so lack for post-test cleanup should not be a problem.

Reply:
Great, done.